sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Infrastructure/BufferedProducerTestConfig.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Messaging.EventHubs.Stress
+{
+    internal class BufferedProducerTestConfig : TestConfig
+    {
+        public int EventEnqueueListSize = 50;
```

Comment:
@jsquire Which member of the options would I use for this? There isn't any significance to the value I chose, I'm just using this value for the size of the event list I'm generating from the `EventGenerator` and passing into `EnqueueEventsAsync` in the `BufferedPublisher`.  Thinking about this more though it would probably be helpful to have this be a dynamic value.

Reply:
oooohhhhh.....  that's my bad.   I interpreted that as the buffer length, not the number of events that we were generating per enqueue call.